Makefile

```
@@ -49,8 +51,8 @@ down-deps: postgres-down hashicorp-down down-networks
 run-acceptance:
 	@go test -v -tags acceptance -count=1 ./tests/acceptance
 
-run-e2e:
-	@go test -v -tags e2e -count=1 ./tests/e2e
+run-e2e: deps
+	@go test -v -tags e2e -count=1 ./tests/e2e ./src/aliases/store
```

why do you need `./src/aliases/store` here?

---

A wrong assumption of what the e2e were, and some tests I ran with real dependencies were in that folder. It moved since, but I forgot to update this.
```suggestion
	@go test -v -tags e2e -count=1 ./tests/e2e
```